ts/data/sample.ts

```diff
@@ -88,12 +88,21 @@ function makeQcStatusColumn(
   };
 }
 
-function makeNameColumn(includeRun: boolean): ColumnDefinition<Sample, void> {
+function makeNameColumn(
+  includeRun: boolean,
+  includeLane?: boolean
```

Comment:
This extra parameter is unnecessary since it should always be the opposite of `includeRun`. If both parameters were true, the lane would be shown twice. There won't be any case where we don't want to show the lane if there is one, so we can just show the lane with the sample name `if (!includeRun && sample.sequencingLane)`